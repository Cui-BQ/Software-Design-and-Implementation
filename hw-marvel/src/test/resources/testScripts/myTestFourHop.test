LoadGraph myTest myTest.tsv

FindPath myTest A F
