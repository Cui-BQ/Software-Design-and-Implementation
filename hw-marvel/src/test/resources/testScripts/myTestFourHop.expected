loaded graph myTest

path from A to F:
A to E via AE
E to D via DE
D to G via DG
G to F via GF